test/test_nn.py

```diff
@@ -211,7 +211,7 @@ def _run_conv2d_same_padding_test(self, BS, C1, C2, H, W, K, S, padding='same',
 
   def test_conv2d_same_padding_odd_input(self):
     BS, C1, H, W = 16, 16, 29, 31
-    C2, K, S, P = 32, 4, 1, 'same'
+    C2, K, S, P = 32, 5, 1, 'same'
```

Comment:
Changing the test is not a good way to fix things

Reply:
The warning was in the test itself. PyTorch throws a warning if same padding is used with even kernel size and odd dilation.